### ISSUE TITLE
Adding a 2.5MB value gives "offset is out of bounds" error

### ISSUE THREAD

**Issue by clibu** [appy-one/acebase#31]
I'm adding a value which is a ` "data:image/jpeg;base64,/...."` which is approximately 2.5MB to AceBase v1.5.0 running in Node v16 and get the following repeatable error:
````
offset is out of bounds at 2021-05-17 14:33:55
        at Uint8Array.set (<anonymous>)
        at c:\webapps\clibuNotes\node_modules\acebase\src\storage-acebase.js:1526:21
        at c:\webapps\clibuNotes\node_modules\acebase\src\storage-acebase.js:1807:41
````
followed by non-stop:
````
read lock on path "/clibu_notes/kvigX6oc4vGkT9V4Gdqy7" by tid 2 (Node.getValue "/clibu_notes/kvigX6oc4vGkT9V4Gdqy7") is taking a long time to
complete [1]
...
````

**Comment by appy-one** [comment]
I've managed to reproduce the issue, and found where this is going wrong. Will be fixed shortly!

**Comment by appy-one** [comment]
Fix published in v1.6.0
